text/0308-cli-advisories.md

```diff
@@ -0,0 +1,280 @@
+                  lambda-layer-node-proxy-agent/layer/package.json.
+         
+        Affected versions: >=1.126.0 <=1.130.0.
+
+        More information at: https://github.com/aws/aws-cdk/issues/17061 
+
+If you don’t want to see an advisory anymore, use "cdk acknowledge ID". For example, "cdk acknowledge 17061".
+```
+
+You can suppress all warnings per individual execution:
+
+```
+$ cdk deploy --no-advisories
+```
+
+And you can disable all advisories indefinitely by adding this entry to
+`~/.cdk.json`:
+
+```
+"advisories": false
+```
+
+Regardless of the state of this flag and the advisories you have acknowledged,
+you can always show the currently active advisories:
+
+```
+$ cdk advisories
+```
+
+This command returns zero if there is no advisory and non-zero otherwise. Users
+can then plug this into a pipeline approval workflow and expect manual review if
+there are any advisories.
+
+> Please note that the acknowledgements are made project by project. If you
+acknowledge an advisory in one CDK project, it will still appear on other 
+projects when you run any CDK commands, unless you have suppressed or disabled 
+advisories.  
+
+### Runbook section (internal to the CDK team)
+
+In case of a high-impact issue, follow these steps:
+
+1. Create or update an issue for this incident on the aws-cdk GitHub repository.
+2. Update the file `advisories.json` on repository `cdklabs/aws-cdk-advisories`, adding an entry for
+   the incident. Example:
+```json
+  {
+    "title":  "Toggling off auto_delete_objects for Bucket empties the bucket",
+    "issueUrl": "https://github.com/aws/aws-cdk/issues/16603",
+    "overview": "If a stack is deployed with an S3 bucket with auto_delete_objects=True, and then re-deployed with auto_delete_objects=False, all the objects in the bucket will be deleted.",
+    "component": ["framework"],
+    "version": "<1.126.0"
+  }
+```
+
+3. Create a PR with this change and wait for an approval. Only SDMs have
+   permission to approve PRs in this repository.
+4. When the PR gets merged, the advisory will be visible to all CLI
+   installations. The GitHub issue will also be automatically updated with the
+   information contained in the file. All the necessary tags will also be added
+   automatically. 
+5. You can keep updating the issue normally, as new information comes in, but
+   you're not allowed to touch the sections auto-generated from the advisories
+   file.
+
+[ ] Signed-off by API Bar Raiser @xxxxx
+
+## Public FAQ
+
+### What are we launching today?
+
+A new communication channel between AWS and users of the CDK. Starting on
+version x.y.z of the CDK CLI, customers will be notified, on every command,
+about security vulnerabilities, regressions and usage of unsupported versions.
+
+### Why should I use this feature?
+
+These advisories shown by the CLI contain very important and actionable
+information about problems that directly affect you. They give you an
```

Comment:
They affect the CDK version you are using, but not necessarily your app.

Maybe for future development we can also publish advisories that match against the specific AWS CDK constructs you are using (we do have that information in the CDK analytics blob).